2023/dotnetvale/data.json

```diff
@@ -0,0 +1,30 @@
+{
+  "ValidMaintainers": {
+    "Description": "GitHub users who maintain this folder. PR for changing this folder will only be accepted from these users.",
+    "GitHubHandles": [
+      "",
+      "",
+      ""
+    ]
+  },
+  "CommunityInfo": {
+    "Name": ".Net Vale",
+    "Country": "Brazil",
+    "Url": "https://www.sympla.com.br/evento/global-azure-bootcamp/1894910?_gl=1*yr5avl*_ga*OTk2NDE2MDUxLjE2NzYxNDc4NzQ.*_ga_KXH10SQTZF*MTY3NzQzMzUwMi4xMS4xLjE2Nzc0MzM3MTYuMC4wLjA",
+    "SessionizeApiId": "10800",
+    "Description_SessionizeApiId": "10800",
```

Comment:
Is this Sessionize ID right? @davidjrh how do I verify this?

Reply:
I put a wrong value.. sorry.. I never used a Sessionize before.. I think that is correct now.

Reply:
Something is wrong. I cannot see any changes!

Reply:
Can you see now? I answered this before the commit, in my github.dev 🤦‍♂️